openhands_cli/agent_chat.py

```diff
@@ -49,8 +49,12 @@
 logger = logging.getLogger(__name__)
 
 
-def setup_agent() -> tuple[LLM | None, Agent | None, Conversation | None]:
-    """Setup the agent with environment variables."""
+def setup_agent() -> tuple[LLM | None, Agent | None, Conversation | None, int]:
```

Comment:
@openhands instead of return an exit code for `setup_up` agent raise an error

once its caught you can throw a return code so the Github CI can catch it